compose.go

```diff
@@ -116,6 +111,21 @@ func NewLocalDockerCompose(filePaths []string, identifier string, opts ...LocalD
 	return dc
 }
 
+// NewLocalDockerCompose returns an instance of the local Docker Compose, using an
+// array of Docker Compose file paths and an identifier for the Compose execution.
+//
+// It will iterate through the array adding '-f compose-file-path' flags to the local
+// Docker Compose execution. The identifier represents the name of the execution,
+// which will define the name of the underlying Docker network and the name of the
+// running Compose services.
+func NewLocalDockerCompose(filePaths []string, identifier string, opts ...LocalDockerComposeOption) *LocalDockerCompose {
+	return NewDockerCompose(filePaths, identifier, LocalDockerComposeExecutor, nil, opts...)
+}
+
+func NewContainerizedDockerCompose(filePaths []string, identifier string, opts ...LocalDockerComposeOption) *LocalDockerCompose {
```

Comment:
Do we still to return a `Local`DockerCompose? It seems weird to me to use a containerised compose method that returns a LocalDC.

I'd be open to using a dedicated DockerisedDockerCompose type, although I think could come with changes in return types and generalisation...

Of course, not a blocker for this PR, but if you are interested in following-up with more refactors to make a stable API, I'm too

Reply:
I started with creating dedicated `DockerisedDockerCompose` type, but got too many duplications. Using current `LocalDockerCompose` with different executors is a best solution for my opinion.

IMHO: interface `DockerCompose` should be removed and type `LocalDockerCompose` should renamed to `DockerCompose`.

P.S.:

I replaced ref to function to interface `DockerComposeExecutor` with 2 implementations.